package.json

```diff
@@ -37,6 +37,7 @@
   },
   "homepage": "https://github.com/mitodl/course-search-utils#readme",
   "dependencies": {
+    "@mitodl/open-api-axios": "^2024.3.22",
```

Comment:
Originally, I was not going to introduce the `@mitodl/open-api-axios`  client in this PR. However, the client previously committed in this repo was missing the new `resource_type: Video` which caused some type errors in MIT Open, so I went ahead and replaced the client in this PR.